scripts/core/superdesk-api.d.ts

```diff
@@ -175,6 +175,10 @@ declare module 'superdesk-api' {
         reinitialize(item: T, profile?: IContentProfileV2): void;
         getValidationErrors(): IAuthoringValidationErrors;
         setValidationErrors(validationErrors: IAuthoringValidationErrors): void;
+        spellchecker: {
+            enabled: boolean;
+            toggleSpellchecker: (nextValue: boolean) => void;
```

Comment:
When a function is called `toggleSomething` it should not accept arguments since toggling means changing to opposite state.

This would be better
`setSpellcheckerStatus: (enabled: boolean) => void;`